python-venv/cmstools/test/cfs_sessions_rc_test/__main__.py

```diff
@@ -0,0 +1,157 @@
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+import argparse
+import sys
+from typing import NoReturn, Callable
+
+from cmstools.test.cfs_sessions_rc_test.helpers.cleanup import cleanup_and_restore
+from cmstools.test.cfs_sessions_rc_test.helpers.setup import get_cfs_config_name
+from cmstools.test.cfs_sessions_rc_test.helpers.setup import cfs_sessions_rc_test_setup
+from cmstools.test.cfs_sessions_rc_test.subtests.cfs_session_gd_base import CFSSessionGDBase
+from cmstools.test.cfs_sessions_rc_test.subtests import cfs_sessions_multi_delete_multi_get_test, \
+    cfs_sessions_multi_delete_test
+from cmstools.lib.log import LOG_FILE_PATH
+
+from .log import logger
+from .defs import ScriptArgs, CFSRCException
+from .argument_parser import (_add_mutually_exclusive_arguments, _add_setup_arguments,
```

Comment:
My understanding is that function/class/variable names starting with underscores should not be imported to other modules. Although it is not enforced, it implies that the method is private/local. If you are intending them to be imported and used, I'd recommend removing the leading underscore.